packages/shared/styles/components/molecules/SfSelect.scss

```diff
@@ -151,6 +150,11 @@
   &--is-required {
     --select-label-required: " *";
   }
+  &--no-chevron {
+    .sf-select__chevron {
```

Comment:
@AdamPawlinski can we use a display variable on '__chevron' and override it here?

Reply:
yes it's obviously better :)